easybuild/easyconfigs/b/bzip2/bzip2-1.0.6-CrayGNU-2015.06.eb

```diff
@@ -11,7 +11,6 @@ toolchainopts = {'pic': True}
 
 source_urls = [
     'https://fossies.org/linux/misc/',
-    'http://www.bzip.org/%(version)s/',
 ]
```

Comment:
@jose-d Since this is now a single entry, it makes sense to reformat this to:

```python
source_urls = ['https://fossies.org/linux/misc/']
```

Are you up for taking care of that?

Reply:
sure.